src/api/spec/features/webui/search_spec.rb

```diff
@@ -262,4 +262,99 @@
       end
     end
   end
+
+  describe 'search for owners' do
+    let(:confirmed_user) { create(:confirmed_user, login: 'Thomas') }
+    let(:other_confirmed_user) { create(:confirmed_user, login: 'Tommy') }
+    let(:group_bugowner) { create(:group, title: 'bugowner_group') }
+    let(:group_maintainer) { create(:group, title: 'maintainer_group') }
+    let(:apache_package) { create(:package, name: 'apache2', title: 'Apache2 package', project: apache) }
+    let(:relationship_package_user) { create(:relationship_package_user, package: apache_package, user: confirmed_user) }
+    let(:relationship_package_group) { create(:relationship_package_group, package: apache_package, group: group_maintainer) }
+    let(:relationship_user_bugowner) { create(:relationship_package_user_as_bugowner, package: apache_package, user: other_confirmed_user) }
+    let(:relationship_group_bugowner) { create(:relationship_package_group_as_bugowner, package: apache_package, group: group_bugowner) }
+    let(:backend_url) { "#{CONFIG['source_url']}/search/published/binary/id?match=(@name='#{apache_package}'+and+(@project='#{apache}'))" }
+    let(:backend_response) { file_fixture('apache_search.xml') }
```

Comment:
interesting, much better than having a `let` block with a hardcoded string. I think I will use it too in another specs